servers/display_server_headless.h

```diff
@@ -134,6 +134,57 @@ class DisplayServerHeadless : public DisplayServer {
+	virtual void screen_set_orientation(ScreenOrientation p_orientation, int p_screen = SCREEN_OF_MAIN_WINDOW) override {}
+	virtual void screen_set_keep_on(bool p_enable) override {}
+
+	virtual void window_set_ime_active(const bool p_active, WindowID p_window = MAIN_WINDOW_ID) override {}
+	virtual void window_set_ime_position(const Point2i &p_pos, WindowID p_window = MAIN_WINDOW_ID) override {}
+
+	virtual void virtual_keyboard_show(const String &p_existing_text, const Rect2 &p_screen_rect = Rect2(), VirtualKeyboardType p_type = KEYBOARD_TYPE_DEFAULT, int p_max_length = -1, int p_cursor_start = -1, int p_cursor_end = -1) override {}
+	virtual void virtual_keyboard_hide() override {}
+
+	virtual void cursor_set_shape(CursorShape p_shape) override {}
+	virtual void cursor_set_custom_image(const Ref<Resource> &p_cursor, CursorShape p_shape = CURSOR_ARROW, const Vector2 &p_hotspot = Vector2()) override {}
+
+	virtual Error dialog_show(String p_title, String p_description, Vector<String> p_buttons, const Callable &p_callback) override { return ERR_UNAVAILABLE; }
+	virtual Error dialog_input_text(String p_title, String p_description, String p_partial, const Callable &p_callback) override { return ERR_UNAVAILABLE; }
+	virtual Error file_dialog_show(const String &p_title, const String &p_current_directory, const String &p_filename, bool p_show_hidden, FileDialogMode p_mode, const Vector<String> &p_filters, const Callable &p_callback) override { return ERR_UNAVAILABLE; }
+	virtual Error file_dialog_with_options_show(const String &p_title, const String &p_current_directory, const String &p_root, const String &p_filename, bool p_show_hidden, FileDialogMode p_mode, const Vector<String> &p_filters, const TypedArray<Dictionary> &p_options, const Callable &p_callback) override { return ERR_UNAVAILABLE; }
+
+	virtual void release_rendering_thread() override {}
+	virtual void make_rendering_thread() override {}
+	virtual void swap_buffers() override {}
+
+	virtual void set_native_icon(const String &p_filename) override {}
+	virtual void set_icon(const Ref<Image> &p_icon) override {}
+
+	virtual IndicatorID create_status_indicator(const Ref<Image> &p_icon, const String &p_tooltip, const Callable &p_callback) override { return INVALID_INDICATOR_ID; }
+	virtual void status_indicator_set_icon(IndicatorID p_id, const Ref<Image> &p_icon) override {}
+	virtual void status_indicator_set_tooltip(IndicatorID p_id, const String &p_tooltip) override {}
+	virtual void status_indicator_set_callback(IndicatorID p_id, const Callable &p_callback) override {}
+	virtual void delete_status_indicator(IndicatorID p_id) override {}
+
+	virtual int64_t window_get_native_handle(HandleType p_handle_type, WindowID p_window = MAIN_WINDOW_ID) const override { return 0; }
+
+	virtual void window_set_vsync_mode(VSyncMode p_vsync_mode, WindowID p_window = MAIN_WINDOW_ID) override {}
+	virtual VSyncMode window_get_vsync_mode(WindowID p_window) const override { return VSyncMode::VSYNC_ENABLED; }
```

Comment:
Should this be `VSYNC_DISABLED`?

Reply:
The fallback value is `VSYNC_ENABLED`: https://github.com/godotengine/godot/blob/029aadef563fb69cf49aa9795b62f27171f8c3f4/servers/display_server.cpp#L748